help/en/package/jmri/jmrit/display/MultiSensorIconAdder.shtml

```diff
@@ -13,6 +13,20 @@
     <div id="mainContent" class="no-sidebar">
       <h1>The MultiSensor Icon Editor</h1>
 
+      <p>A MultiSensor Icon lets you display one of N icons, 
+        depending on which one of N associated sensors is active.
+        For example, you could have three icons displaying the three positions of
+        a CTC lever; see image below.  But it doesn't have to be three.  
+        For example, you could have ten icons representing the positions of a
+        thumbwheel switch.</p>
+      <p>The user can move to the next or previous icon by
+        clicking on the icon.  This will deactivate the current sensor
```

Comment:
"clicking on the icon" in one of two specific regions of the icon as described below.

Reply:
Thanks.  I've added some to this sentence.